docker-compose.yaml

```diff
@@ -1,8 +1,10 @@
 version: '2'
 services:
   phpschool:
+    container_name: phpschool
     build: ./
     volumes:
-      - .:/phpschool/projectfiles:rw
+      - ./code:/phpschool/code:rw
     environment:
       TERM: xterm
+    command: tail -f /dev/null
```

Comment:
It's a bit weird but it ensures the container will stay running. This is the Windows thing we're compensating for (as mentioned in the desc) 

We should be able to drop it once docker compose releases 1.9 and just have that as the required version imo
This is their [fix](https://github.com/docker/compose/pull/3980/files) which just hit `master` 🎉